airootfs/usr/share/vx-img/scrape-image.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+IFS=$'\n' read -r -d '' -a disks <<< "$(lsblk -x SIZE -nblo NAME,LABEL,SIZE,TYPE | grep "disk" | awk '{print $1}')"
+
+_disk="/dev/nvme0n1"
+_diskname=""
+
+clear
+echo "Found the following disks to scrape:"
+while true; do
+
+    # Get a list of all available disks large enough to take our image, sorted by size
+    IFS=$'\n' read -r -d '' -a disks <<< "$(lsblk -x SIZE -nblo NAME,SIZE,TYPE | grep "disk" | awk '{print $1}')"
+
+    # Get the sizes of all these disks
+    IFS=$'\n' read -r -d '' -a sizes <<< "$(lsblk -x SIZE -nblo NAME,SIZE,TYPE | grep "disk" | awk '{print $2}')"
```

Comment:
I'm unfamiliar with this style of bash scripting, using `read` but then feeding input straight to it. That said, I'm far from the world's expert on bash. If this is standard, cool. If not, I do find it a bit confusing and may be worth something a little more straightforward to read?

Any way to get both data fields in one call rather than the same call twice?